lte/gateway/deploy/roles/ovs_deploy/tasks/main.yml

```diff
@@ -11,69 +11,112 @@
 # limitations under the License.
 
 
-- fail: msg="PACKAGE_LOCATION is required"
-  when:
-    - PACKAGE_LOCATION is not defined
-
-- name: Include vars of all.yaml.
+- name: Include vars of all.yaml
   include_vars:
     file: all.yaml
-    name: all_vars
 
-- name: Converting PACKAGE_LOCATION to absolute path
-  shell: realpath {{ PACKAGE_LOCATION }}
-  register: PACKAGE_LOCATION_ABSOLUTE
+- name: Configuring private package manager.
+  copy:
+    content: >
+      deb {{ ovs_pkgrepo_proto }}://{{ ovs_pkgrepo_host }}{{ ovs_pkgrepo_path}}
```

Comment:
I would read that from `all.yaml`

Reply:
default values are in `all.yml`, may be overridden from ansible command line (see above)